objects.js

```diff
@@ -2634,96 +2634,149 @@ SpriteMorph.prototype.searchBlocks = function (
     if (searchString) {searchPane.reactToKeystroke(); }
 };
 
-// SpritMorph parsing simple arithmetic expressions to reporter blocks
+// SpriteMorph parsing simple arithmetic expressions to reporter blocks
 
 SpriteMorph.prototype.reporterize = function (expressionString) {
     // highly experimental Christmas Easter Egg 2016 :-)
     var ast;
 
-    function parseInfix(expression, operator, already) {
-        // very basic diadic infix parser for arithmetic expressions
-        // with strict left-to-right operator precedence (like in Smalltalk)
-        // which can be overriden by - nested - parentheses.
-        // assumes well-formed expressions, no graceful error handling yet.
-
-        var inputs = ['', ''],
-            idx = 0,
-            ch;
-
-        function format(value) {
-            return value instanceof Array || isNaN(+value) ? value : +value;
-        }
-
-        function nested() {
-            var level = 1,
-                expr = '';
-            while (idx < expression.length) {
-                ch = expression[idx];
-                idx += 1;
-                switch (ch) {
-                case '(':
-                    level += 1;
-                    break;
-                case ')':
-                    level -= 1;
-                    if (!level) {
-                        return expr;
-                    }
-                    break;
-                }
-                expr += ch;
+    function tokenize(expressionString) {
+        // very basic tokenizer for arithmetic expressions
+        var tokens = [];
```

Comment:
These should be:
```js

var a = 1,
    b = 2,
    c = 3;
```
formatted (to be consistent with Snap<em>!</em>)

Also, I really feel like `IDENT` should just be `IDENTIFIER` since we're not really short on characters.